src/app/calendar-header/calendar-header.component.html

```diff
@@ -17,47 +17,61 @@
 </main>
 
 
-<div id="go-live" class="modal bottom-sheet">
+<div id="go-live" class="modal">
```

Comment:
Commenting here for future discussion. Was the bottom-sheet modal causing issues?